sections/units.tex

```diff
@@ -19,7 +19,7 @@
 \textbf{Reinforcing} requires that your Town has a Citadel in addition to a Dwelling of that Unit's Level.\par
 
 \note{13}{
-  Any effects which allow you to Reinforce outside of using the Population Token \textbf{do not} require a Citadel or any Dwellings.\par
+  Any effects which instruct you to Reinforce \textbf{do not} require using up the Population Token nor owning a Citadel or any Dwellings.\par
```

Comment:
"Using up" feels a little like slang. How about "flipping"? Or "turning over"? Is there a formal term for this?

Reply:
Just remembered, Deemer spends a token 😄